coordinator/src/main/java/org/apache/uniffle/coordinator/web/servlet/admin/RefreshCheckerServlet.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.uniffle.coordinator.web.servlet.admin;
+
+import java.util.List;
+import java.util.Optional;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.collections.CollectionUtils;
+
+import org.apache.uniffle.coordinator.CoordinatorConf;
+import org.apache.uniffle.coordinator.CoordinatorServer;
+import org.apache.uniffle.coordinator.access.checker.AccessChecker;
+import org.apache.uniffle.coordinator.web.Response;
+import org.apache.uniffle.coordinator.web.servlet.BaseServlet;
+
+
+public class RefreshCheckerServlet extends BaseServlet {
+  private final CoordinatorServer coordinator;
+
+  public RefreshCheckerServlet(CoordinatorServer coordinator) {
+    this.coordinator = coordinator;
+  }
+
+  @Override
+  protected Response handleGet(HttpServletRequest req, HttpServletResponse resp) {
+    List<String> checkers = coordinator.getCoordinatorConf().get(CoordinatorConf.COORDINATOR_ACCESS_CHECKERS);
+    String accessClass = req.getParameter("class");
+    if (CollectionUtils.isEmpty(checkers)) {
+      return Response.fail("Access checkers is empty, will not update any checkers.");
+    }
+    try {
+      Class<?> aClass = Class.forName(accessClass);
```

Comment:
What if accessClass is empty? Should we refresh all checkers?

Reply:
> What if accessClass is empty? Should we refresh all checkers?

Updating all accessClass configurations is a good idea, as it can reduce the amount of class input required by the user.